### PR TITLE
Apm update

### PR DESCRIPTION
**Pull request by l0k0ms**
1 - Removing APM help section since it's a duplicate of the main help section.

2 - Adding disclaimer for python instrumentation.
A customer had trouble to setup its APM monitoring, and no traces were received.
After some time, they noted that tracing was disabled when the application was run in DEBUG mode. 
This is apparently not clear enough, and can lead to issues as this one
Preview: https://d2cx6t4ssmwdrv.cloudfront.net/gus/apm-faq-update/tracing/python/

3 - Adding a Faq section for agent submit frequence
Preview: https://d2cx6t4ssmwdrv.cloudfront.net/gus/apm-faq-update/tracing/faq/#how-frequent-does-the-trace-agent-send-stats

4 - Adding docker specific documentation to make it easier for customers to send trace with docker
Preview: https://d2cx6t4ssmwdrv.cloudfront.net/gus/apm-faq-update/tracing/docker/

### CODE REVIEW

File: content/tracing/python.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ pip install ddtrace
 
 Finally, import the tracer and instrument your code!
 
+<div class="alert alert-warn">
+For Python applciations, please note that tracing is disabled when your application is launched in <b>DEBUG</b> mode.
+Find more <a href="http://pypi.datadoghq.com/trace/docs/#module-ddtrace.contrib.django">here</a>
```
Comment (technovangelist) [inline content/tracing/python.md]: this sounds like something is missing. Find more what here?? perhaps "find more details here" or "learn more about this here" or something like that

File: content/tracing/faq.md [hunk 2]
```diff
@@ -19,6 +19,10 @@ customnav: tracingnav
 
 Tracing data is stored for about 36 hours.
 
+### How frequent does the trace-agent send stats?
```
Comment (technovangelist) [inline content/tracing/faq.md]: hmmm, sounds strange to me. maybe "How often does the trace agent send stats?"

File: content/tracing/index.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ Install the latest [Datadog Agent](https://app.datadoghq.com/account/settings#ag
 
 To trace applications in Docker containers, you can use the [docker-dd-agent](https://hub.docker.com/r/datadog/docker-dd-agent/) image (tagged version 11.0.5110 or higher) and enable tracing by passing `DD_APM_ENABLED=true` as an environment variable.
 
-For additional information, please reference [the project README on Github](https://github.com/DataDog/docker-dd-agent/blob/master/README.md#tracing--apm)
+For additional information, please reference [the dedicated page](/tracing/docker)
```
Comment (technovangelist) [inline content/tracing/index.md]: the wording made me think of kasey casem dedicating a song to some listener on america's top something (some show that died after a long time). "please reference the Docker page"

File: content/tracing/faq.md (hunk 2 above)
Comment (l0k0ms) [reply content/tracing/faq.md]: 👍

File: content/tracing/index.md (hunk 3 above)
Comment (l0k0ms) [reply content/tracing/index.md]: 👍

**Pull request merged**